internal/storage/storage_handle_test.go

```diff
@@ -34,6 +34,7 @@ func getDefaultStorageClientConfig() (clientConfig StorageClientConfig) {
 		HttpClientTimeout:   800 * time.Millisecond,
 		MaxRetryDuration:    30 * time.Second,
 		RetryMultiplier:     2,
+		UserAgent:           "AppName",
```

Comment:
Can we set some close to real-world UserAgent ?

Reply:
fixed